tools/docker/boardfarm-ci/Dockerfile

```diff
@@ -1,42 +1,30 @@
 FROM python:3.8-slim-buster
 
+# Debian dependencies to install docker and docker compose from docker.com
 RUN apt-get update \
-    && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
-    apt-transport-https \
-    ca-certificates curl \
-    curl \
-    gcc \
-    git \
-    gnupg \
-    gnupg-agent \
-    libsnmp-dev \
-    netcat \
-    software-properties-common \
-    wireshark-common \
-    && rm -rf /var/lib/apt/lists/*
-
-COPY requirements.txt /app/requirements.txt
-
-WORKDIR app
-RUN pip3 install -r requirements.txt
-
-# TODO: what needs this?
-#RUN pip3 install jsonschema distro
-
+&& DEBIAN_FRONTEND=noninteractive apt-get install \
+apt-transport-https ca-certificates curl gnupg-agent \
+software-properties-common curl gcc libsnmp-dev wireshark-common tshark \
+-y && apt-get clean
```

Comment:
Do we still need wireshark and tshark inside the boardfarm-ci image? We already know that sniffer doesn't work when we try that way. Maybe we can keep it and I'll exclude it in the PR fixing PPM-331.

Reply:
Yes, I can think it is better removed on the same branch that is depending on having dumpcap installed alongside python 3.5 on the host for dctest to work (and checking the dependency -- that the executable exists -- as it is done for docker and docker-compose)